test/e2e/framework/node/resource.go

```diff
@@ -583,6 +583,27 @@ func GetSchedulableClusterZones(c clientset.Interface) (sets.String, error) {
 	return zones, nil
 }
 
+func CheckNodesAllocatableCPUsEquivalent(c clientset.Interface, nodeNames []string) (bool, error) {
+	var lastNodeCPU resource.Quantity
+	for i, nodeName := range nodeNames {
+		node, err := c.CoreV1().Nodes().Get(context.TODO(), nodeName, metav1.GetOptions{})
+		if err != nil {
+			return false, fmt.Errorf("Error when getting a Node with name '%s'", nodeName)
+		}
```

Comment:
you can do a List() and then filter out the nodes that are not in the nodeNames array

Reply:
done